WTWSMS/history/titles/k_nicene.txt

```diff
@@ -0,0 +1,14 @@
+37.1.1={
+	
+	liege="e_byzantium"
+	
+
+	active = yes
+
+	
+
+	government = theocracy_government
+	
+	succession = succ_open_elective
+
+}
```

Comment:
That was a bug I discovered with both Investiture religions AND Pentarchies.

Basically, what happens is that the big "Main Religious Head" icon on the far left of the Religion screen is blank, while the religious head is on the list of Pentarchs, with "Pentarch" as his title. It removes a religious head.

This happened to the Miaphysites, Nicenes, and Priscillians several times. After testing, I discovered that forcing Open Elective Succession in their history always fixed the problem. This was a bug fix. It is simple as that.